requery-test/build.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
     testCompile 'javax.cache:cache-api:1.0.0'
     compile "com.fasterxml.jackson.core:jackson-annotations:$jackson_version"
     compile 'junit:junit:4.12'
+    compile "com.google.guava:guava:20.0"
```

Comment:
Since you only use BiMap in your test this isn't really worth bringing all of guava in. Could you do this with a regular Map? thanks